ts/screens/onboarding/EmailInsertScreen.tsx

```diff
@@ -295,8 +290,7 @@ class EmailInsertScreen extends React.PureComponent<Props, State> {
                     autoCapitalize: "none",
                     keyboardType: "email-address",
                     value: this.state.email.getOrElse(EMPTY_EMAIL),
-                    onChangeText: this.handleOnChangeEmailText,
-                    style: styles.emailInput
```

Comment:
It's corrects that the weight of the word is different?
| Before | After|
|-------|-------|
<img src="https://user-images.githubusercontent.com/11773070/126491052-bfdeb6df-3338-424f-bb18-f4b90e76fd58.png" width=200>|<img src="https://user-images.githubusercontent.com/11773070/126491127-9651c8de-081f-4290-8c29-a6020afd3a98.png" width=200>|

Reply:
I also talked about this case with @thisisjp, I don't remember the discussion. If I'm not mistaken, here too he has chosen to maintain the same behaviour as the other inputs and therefore change the font weight, I ask for his intervention to clarify the doubts.

Reply:
correct, this story is a huge effort of 'normalizing' different UI\behaviours collected throughout the years